_docs/quickstarts/forseti-security/index.md

```diff
@@ -39,63 +67,28 @@ file based on your input.
       cd forseti-security/scripts/gcp_setup
       ```
 
-  1. Start the setup process:
+  1. Invoke the setup:
   
       If you downloaded a certain release of Forseti, specify the release version
       to the setup wizard.
-
-      Setup wizard is not available prior to 1.1.0.
-      
-      _e.g. Deploy 1.1.0:_
-      
-      ```bash
-      python setup_forseti.py --version 1.1.0
-      ```
-  
-     _Default: runs master branch_
      
       ```bash
```

Comment:
I agree with Michael, this section reads as if I should run all these commands together. Ideally they'd be separated into chunks with context for each command, something along these lines:

```
1. To start the setup process for the default release version, run `python setup_forseti.py`.
  2. To specify a Forseti release version, add the version flag `-- version #.#.#`.
To display help information, run `python setup_forseti.py -h`.
```